host_vars/sn06.galaxyproject.eu.yml

```diff
@@ -1,9 +1,9 @@
 ---
 htcondor_network_interface: ens802f0.223
 
-# Settings specific to the `usegalaxy_eu.htcondor` role.
-condor_daemons:
-  - COLLECTOR
-  - NEGOTIATOR
-  - MASTER
-  - SCHEDD
+# 15/03/2024: On sn06 HTCondor conf was manually adjusted to use port 9618 since the
```

Comment:
We can stop all job submissions on the Galaxy side via the Admin panel. So no official downtime is needed? Or we could simply stop all job-handlers?

Reply:
We can stop the handlers and restart the htcondor systemd service.